pkg/scheduler/scheduler.go

```diff
@@ -379,10 +388,14 @@ var defaultQueueingHintFn = func(_ klog.Logger, _ *v1.Pod, _, _ interface{}) (fr
 	return framework.Queue, nil
 }
 
-func buildQueueingHintMap(es []framework.EnqueueExtensions) internalqueue.QueueingHintMap {
+func buildQueueingHintMap(ctx context.Context, es []framework.EnqueueExtensions) (internalqueue.QueueingHintMap, error) {
```

Comment:
can you add one test case  to `Test_buildQueueingHintMap`, which an `EventsToRegister` returns error?

Reply:
done